src/goto-programs/html_goto_trace.cpp

```diff
@@ -0,0 +1,278 @@
+/*******************************************************************\
+
+Module: HTML Traces of GOTO Programs
+
+Author: elizabeth.polgreen@cs.ox.ac.uk
+
+  Date: October 2017
+
+\*******************************************************************/
+
+/// \file
+/// HTML traces of GOTO Programs
+
+#include "goto_trace.h"
+
+#include <cassert>
+#include <ostream>
+
+#include <util/arith_tools.h>
+#include <util/config.h>
+#include <util/symbol.h>
+
+#include <ansi-c/printf_formatter.h>
+#include <langapi/language_util.h>
+
+
+void output_html_header(std::ostream &out)
+{
+  out << "<!DOCTYPE html> \n" << "<html> \n" << "<head> \n" << "<style> \n"
+      << "button.function_call{ background-color: #eee; color: Black; cursor: pointer; \n"  // NOLINT(*)
```

Comment:
Suggest a break after each property to make this easier on people hand-editing the HTML or editing this code